docs/source/spatialstats.rst

```diff
@@ -0,0 +1,108 @@
+- quantify the spatial correlation of measurement errors in DEMs (e.g., native spatial resolution, instrument noise),
+- estimate robust errors for observations integrated in space (e.g., average or sum of samples),
+- propagate errors between spatial ensembles at different scales (e.g., sum of glacier volume changes).
+
+.. contents:: Contents 
+   :local:
+
+Assumptions for statistical inference in spatial statistics
+***********************************************************
+
+Spatial statistics are valid if the variable of interest verifies the assumption of stationarity of the 1\ :sup:`st` and 2\ :sup:`nd` orders.
+That is, if the two following assumptions are verified:
+
+1. The mean of the variable of interest is stationary in space, i.e. constant over sufficiently large areas,
+2. The variance of the variable of interest is stationary in space, i.e. constant over sufficiently large areas.
+
+A sufficiently large averaging area is an area expected to fit within the spatial domain studied.
+
+In other words, for a reliable analysis, the DEM should:
+
+1. Not contain systematic biases that do not average out over sufficiently large distances (e.g., shifts, tilts), but can contain pseudo-periodic biases (e.g., along-track undulations),
+2. Not contain measurement errors that vary significantly in space.
+
+Precision of a single DEM, or of a difference of elevation data
+***************************************************************
+
+To infer the precision of a DEM, it is compared against other elevation data.
+If the other elevation data is known to be of higher-precision, one can assume that the analysis of differences will represent the precision of the rougher DEM.
+Otherwise, the difference will describe the precision with significant measurement errors originating from both the DEM and the other dataset.
+
+TODO: complete with Hugonnet et al. (in prep)
+
+Stable terrain: proxy for infering DEM precision
+************************************************
+
+TODO
+
+Metrics for DEM precision
+*************************
+
+Pixel-wise elevation measurement error
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+TODO
+
+
+Spatially-integrated elevation measurement error
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The standard error (SE) of a statistic is the standard deviation of the distribution of this statistic.
+For spatially distributed samples, the standard error of the mean (SEM) is of great interest as it allows quantification of the error of a mean (or sum) of samples in space.
+
+The standard error  :math:`\sigma_{\overline{dh}}` of the mean :math:`\overline{dh}` of elevation changes samples :math:`dh` is typically derived as:
+
+.. math::
+
+        \sigma_{\overline{dh}} = \frac{\sigma_{dh}}{\sqrt{N}},
+
+where :math:`\sigma_{dh}` is the dispersion of the samples, and :math:`N` is the number of **independent** observations.
```

Comment:
Here is the first time that the word "dispersion" is used. Would be nice with an explanation of its meaning and/or a link to more information (such as wikipedia?)

Reply:
We should call it by its name, i.e. standard deviation!

Reply:
Ok, I guess this is to be more generic in case we use NMAD etc.
Then I think the term is statistical dispersion, but dispersion or spread are synonyms: https://en.wikipedia.org/wiki/Statistical_dispersion

Reply:
Indeed, the standard deviation is only a measure of the dispersion, and the mean a measure of central tendency.
Because we use robust statistics, we'll have to specify that we tend to use the median/50th percentile or the NMAD/68th absolute percentile as measures of central tendency and dispersion, rather than the typical mean/std.

This should be clearly introducted in a few words. Maybe in the robust statistics documentation, and point there from the spatial statistics documentation when a the term "dispersion" is brought up?

Reply:
Sounds good!